test/extended/cluster/cl.go

```diff
@@ -0,0 +1,182 @@
+package cluster
+
+import (
+	"fmt"
+	"path/filepath"
+	"strconv"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+
+	oapi "github.com/openshift/origin/pkg/api"
+	projectapi "github.com/openshift/origin/pkg/project/api"
+	exutil "github.com/openshift/origin/test/extended/util"
+	testutil "github.com/openshift/origin/test/util"
+	kapi "k8s.io/kubernetes/pkg/api"
+	kclientset "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
+)
+
+const deploymentRunTimeout = 5 * time.Minute
+
+var _ = g.Describe("[Feature:Performance][Serial][Slow] Load cluster", func() {
+	defer g.GinkgoRecover()
+	var (
+		oc = exutil.NewCLI("cl", exutil.KubeConfigPath())
+		_  = exutil.FixturePath("testdata", "cluster", "master-vert.yaml")
+	)
+
+	var c *kclientset.Clientset
+	g.BeforeEach(func() {
+		c = oc.AdminKubeClient()
+		ParseConfig(e2e.TestContext.Viper)
+	})
+
+	g.It("should load the cluster", func() {
+
+		project := ConfigContext.ClusterLoader.Projects
+		tuningSets := ConfigContext.ClusterLoader.TuningSets
+		if len(project) < 1 {
+			e2e.Failf("invalid config file.\nFile: %v", project)
+		}
+
+		var namespaces []string
+		//totalPods := 0 // Keep track of how many pods for stepping
+		// TODO sjug: add concurrency
+		testStartTime := time.Now()
+		for _, p := range project {
```

Comment:
Instead of having this all in one test, I'd like each project to be its own test so that we can filter on it:

```
for _, p := range project {
  g.It(fmt.Sprintf("with %s", p.Basename), func() {
    ...
  })
}
```

That way we can run a single project